packages/terra-toggle/src/Toggle.jsx

```diff
@@ -26,85 +28,30 @@ const defaultProps = {
   isOpen: false,
 };
 
-class Toggle extends React.Component {
-  constructor(props) {
-    super(props);
-    this.handleOnAnimationEnd = this.handleOnAnimationEnd.bind(this);
-    this.handleOnAnimationStart = this.handleOnAnimationStart.bind(this);
-    this.animationDuration = 250;
-  }
-
-  componentDidMount() {
-    // If toggle is closed when the component mounts, set props on all the focusable elements
-    // within the toggle to disable users from focusing to them
-    if (!this.props.isOpen) {
-      if (this.contentContainer) {
-        this.contentContainer.classList.add('is-closed');
-      }
-    }
-  }
-
-  handleOnAnimationEnd(open) {
-    // If toggle is closed and the close animation has ended, set props on all the focusable elements
-    // within the toggle to disable users from focusing to them
-    if (!open) {
-      if (this.contentContainer) {
-        setTimeout(() => this.contentContainer.classList.add('is-closed'), this.animationDuration);
-      }
-    }
-  }
-
-  handleOnAnimationStart(open) {
-    // If toggle is opened and the open animation has started, remove props on all the focusable elements
-    // within the toggle that disable users from focusing to them
-    if (open) {
-      if (this.contentContainer) {
-        this.contentContainer.classList.remove('is-closed');
-      }
-    }
-  }
-
-  render() {
-    const { isAnimated, isOpen, children, ...customProps } = this.props;
-    const attributes = Object.assign({}, customProps);
-    const ToggleClassNames = classNames([
-      'terra-Toggle',
-      { 'is-animated': isAnimated },
-      attributes.className,
-    ]);
-    const height = isOpen ? 'auto' : '0';
-    let body;
+const Toggle = ({ isAnimated, isOpen, children, ...customProps }) => {
+  const height = isOpen ? 'auto' : '0';
+  let body;
 
-    if (isAnimated) {
-      body = (
-        <div className="terra-Toggle-content" ref={(div) => { this.contentContainer = div; }}>
-          <AnimateHeight
-            duration={this.animationDuration}
-            height={height}
-            onAnimationEnd={this.handleOnAnimationEnd(isOpen)}
-            onAnimationStart={this.handleOnAnimationStart(isOpen)}
-          >
-            {children}
-          </AnimateHeight>
-        </div>
-      );
-    } else {
-      body = (
-        isOpen && children
-      );
-    }
-
-    return (
-      <div
-        {...attributes}
-        className={ToggleClassNames}
-        aria-hidden={!isOpen}
-      >
-        {body}
-      </div>
+  if (isAnimated) {
+    body = (
```

Comment:
Did you intended to remove this div
`<div className="terra-Toggle-content" ref={(div) => { this.contentContainer = div; }}>` from around the animated body?